schema.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+"""Describes raw data table schema in terms of SQL Alchemy MetaData objects.
+
+The `create_schema()` function can be used to generate raw data table formats, 
+for easy loading/dumping to SQLite.
+
+TODO:
+    * review schema for any missing attributes/indices 
+    * unit tests for schema verification (?)
+"""
+from sqlalchemy import Float, String, Boolean, Integer, create_engine
+from sqlalchemy.schema import Column, MetaData, Table, Sequence, Index
+
+def create_schema():
+    """Constructs SQL Alchemy MetaData and defines raw data table format.
+
+    See the SQL Alchemy docs for MetaData attributes and manipulation: 
+        https://docs.sqlalchemy.org/en/13/core/metadata.html
+
+    Returns:
+        sqlalchemy.MetaData: MetaData object ready for inspection or connection to database engine.
+
+    Examples:
+        Create an in-memory sqlite db and list table names
+        
+        >>> engine = create_engine('sqlite:///:memory:')    
+        >>> schema = create_schema()
+        >>> schema.create_all(engine)
+        >>> for t in schema.sorted_tables: print(t.name)
+        applications_foreground
+        battery
+        calls
+        light
+        locations
+        messages
+        screen
+
+    TODO:
+        * does device_id need the max character limit?
```

Comment:
we could put it as varchar(255)

Reply:
I don't think VARCHAR(N) is a thing in sqlite. I think everything is text, even if it's specified as a sized varchar

Reply:
Resolved in 04f1f1f